File: CS50x/sentimental-mario-more/mario.py
# TODO
while True:
    height = input("Height: ")
    if height.isdigit():
        height = int(height)
        if height >= 1 and height <= 8:
            break

for i in range(height):
    for j in range(height-1-i):
        print(" ", end="")

    for k in range(i+1):
        print("#", end="")

    print("  ", end="")

    for l in range(i+1):
        print("#", end="")

    print()

File: CS50x/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        // Input of height of pyramid
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //1st loop to print the row
    for (int i = 0; i < height; i++)
    {
        //2nd loop to give space
        for (int k = 0; k < height - i - 1; k++)
        {
            printf(" ");
        }
        //3rd loop to print the hash
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("  ");

        //4th loop to print hash again after a gap
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }

        printf("\n");

    }
    return 0;
}